src/currencyservice/server.js

```diff
@@ -141,9 +139,7 @@ function _carry (amount) {
 function getSupportedCurrencies (call, callback) {
   // Extract the span context received from the gRPC client.
   // Create a child span and add an event.
-  const currentSpan = tracer.getCurrentSpan();
   const span = tracer.startSpan('currencyservice:GetSupportedCurrencies()', {
-    parent: currentSpan,
```

Comment:
is the new API automatically associating the new span with its parent?

Reply:
I did some testing and it looks like the spans are captured correctly in the Cloud Console, but it would be good if someone with more OT experience could verify

Reply:
Looking at more documentation, it does seem like it is supposed to capture the parent trace from the environment. But just to be safe, I added back in an explicit parent using the new API